contracts/context-config/src/query.rs

```diff
@@ -44,6 +44,15 @@ impl ContextConfigs {
         members
     }
 
+    pub fn has_member(&self, context_id: Repr<ContextId>, identity: Repr<ContextIdentity>) -> bool {
+        let context = self
+            .contexts
+            .get(&context_id)
+            .expect("context does not exist");
```

Comment:
We may want to think about returning a `Result` here.

Reply:
I followed the structure of the other functions in the module. I had to add this helper to faciliate the cx contract call. cc: @miraclx

Reply:
yeah, so NEAR contracts will only roll back a failed transaction if execution fails, and they fail only when the contract panics, unlike ours.

Returning a result will serialize the result as a success value, even if it's an `Err`, 🫠 

So the spent gas won't be refunded, state mutations will hold, etc..